src/middleware/jirahost-middleware.test.ts

```diff
@@ -40,17 +40,6 @@ describe("await jirahostMiddleware", () => {
 			expect(next).toBeCalled();
 		});
 
-		it("cookie should have priority", async () => {
```

Comment:
Deleted this because it's not needed now that I've changed the jirahost-middleware to set the jirahost from req.body on all methods except `GET`